cmd/resources/xrootd.cfg

```diff
@@ -14,43 +14,43 @@
 # limitations under the License.
 #
 
-all.manager {{.ManagerHost}}+ {{.ManagerPort}}
+all.manager {{.Xrootd.ManagerHost}}+ {{.Xrootd.ManagerPort}}
 all.role server
 if exec xrootd
-  xrd.port {{.Port}}
-  xrd.protocol http:{{.Port}} libXrdHttp.so
+  xrd.port {{.Xrootd.Port}}
+  xrd.protocol http:{{.Xrootd.Port}} libXrdHttp.so
 fi
-xrd.tls {{.TLSCertificate}} {{.TLSKey}}
-{{if .TLSCertDir}}
-xrd.tlsca certdir {{.TLSCertDir}}
+xrd.tls {{.Server.TLSCertificate}} {{.Server.TLSKey}}
+{{if .Xrootd.TLSCertDir}}
+xrd.tlsca certdir {{.Xrootd.TLSCertDir}}
 {{else}}
-xrd.tlsca certfile {{.TLSCertFile}}
+xrd.tlsca certfile {{.Xrootd.TLSCertFile}}
```

Comment:
Similarly, rename this `Server.TLSCACertificateFile`.  It's not symmetric with the host certificate (`TLSCertificate`) but I find it useful to keep the symmetry of the file and directory.